src/index.js

```diff
@@ -1,8 +1,7 @@
-import window from 'global/window';
-import console from 'global/console';
-
+import {window, console} from '../helpers/global/index.js';
+import {get} from '../node_modules/@polymer/polymer/lib/utils/path.js'
```

Comment:
Is there a reason why we aren't just importing `@polymer/polymer/lib/utils/path.js`. Rollup/Webpack will resolve this. Is this a browser module specific import?

Reply:
I am still testing this. I haven't seen any other packages with 3rd party dependencies, therefore I will do some testing. 

It is in fact a browser import. If this is packed via rollup we would load polymer(`Path.get`) twice in our projects.

Lets just keep this open for a while. I might have a cleaner option.